nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -119,6 +119,8 @@
 
 - `linuxPackages.nvidiaPackages.dc_520` has been removed since it is marked broken and there are better newer alternatives.
 
+- `programs.less.lessopen` is now null by default. To restore the previous behaviour, set it to `''${lib.getBin pkgs.lesspipe "lesspipe.sh"} %s''`.
```

Comment:
```suggestion
- `programs.less.lessopen` is now null by default. To restore the previous behaviour, set it to `''${lib.getExe' pkgs.lesspipe "lesspipe.sh"} %s''`.
```

Reply:
Good catch on the `|` too, thanks @eclairevoyant and @FliegendeWurst